joomla/joomla-cms
# [postgresql] Fix SQL errors in schema updates which make update from 3.8.12 to staging fail

Pull Request for new issue.

### Summary of Changes

1. Add missing column `title_holder` to columns list in insert statement for table `#__action_log_config` in schema update `3.9.0-2018-05-05.sql` for PostgreSQL to fix SQL error about missing column `title_holder` on Joomla update.
2. Correct text quoting in schema update `3.9.0-2018-06-02.sql` for PostgreSQL to fix SQL error about missing column `Article` on Joomla update. 
3. Correct names quoting in schema update `3.9.0-2018-07-09.sql` for PostgreSQL to fix SQL error near "´".

Without this PR, update from 3.8.12 to 3.9-beta2-dev nightly build from today fails on PostgreSQL with SQL errors.

With this PR, update from 3.8.12 to 3.9-beta2-dev nightly build from today plus this PR applied finish with success.

### Testing Instructions

Code review:
- Issues 1. and 2.: Verify the SQL statements corrected by this PR are consistent with the corresponding statements in the same schema updates for MySQL.
- Issue3.: Check that correction in schema update `3.9.0-2018-07-09.sql` for PostgreSQL is correct.

Real test:
1. Update a 3.8.12 to the latest 3.9-beta2-dev nightly build from today. Result: See actual result below.
2. Download the update package of the latest 3.9-beta2-dev nightly build from today, unpack it, apply the changes from this PR here, pack it again and update a 3.8.12 to that modified package using the package upload method. Result: See expected result below.

### Expected result

Code review:
1. Both statements in schema update `postgresql/3.9.0-2018-05-05.sql` have same columns in their column list because they both insert data for the same columns.
2. `WHERE "type_title" = 'Article'` is quoted correctly in schema update `postgresql/3.9.0-2018-06-02.sql`.
3. Names are correctly quoted in schema update `postgresql/3.9.0-2018-07-09.sql`.

Real test:
Update finish with success.

### Actual result

Code review:
1. In the schema update `postgresql/3.9.0-2018-05-05.sql` for PostgreSQL the column `title_holder` is missing in the columns list of the insert statement for table `#__action_log_config`. This causes an SQL error when updating a 3.8.12 to the night build of 3.9-beta2-dev from today.
2. `WHERE "type_title" = "Article"` is not quoted correctly in schema update `postgresql/3.9.0-2018-06-02.sql`.
3. Names are quoted in the MySQL way in schema update `postgresql/3.9.0-2018-07-09.sql`.

Real test:
Update breaks with SQL errors.

### Documentation Changes Required

None.
